src/common_types.hpp

```diff
@@ -57,12 +57,12 @@ using time_type = float;
 // group equal kinds in the same cell group.
 
 enum cell_kind {
```

Comment:
I would remove the unimplemented kinds, i.e. leave just `cable1d_neuron`.
Others can be added as they are implemented.